core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2258,6 +2258,10 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-azure-qe
     namespace: ci
+  - cluster_groups:
+    - non_app_ci:
```

Comment:
i don't think there should be a `:` at the end of this line?

Reply:
Good catch, updated the logic!